packages/sdk-js/src/types/growthbook.ts

```diff
@@ -204,14 +224,19 @@ export interface Context {
   streamingHost?: string;
   apiHostRequestHeaders?: Record<string, string>;
   streamingHostRequestHeaders?: Record<string, string>;
+  payload?: StoredPayload;
```

Comment:
move payload to feature repo, keyed. make hydration pass the payload through to the feature repo instead of living on each sdk instance??